### ISSUE TITLE
best.pt单独evaluate的mAP比训练过程中得到的mAP低

### ISSUE THREAD

**Issue by April4lu** [argusswift/YOLOv4-pytorch#122]
答主您好，在训练YOLOv4过程中，best_test_mAP为0.614
![image](https://user-images.githubusercontent.com/32457231/103716232-8153b900-4ffd-11eb-9e12-ee644dc3d69c.png)
但是执行`CUDA_VISIBLE_DEVICES=0 python3 eval_voc.py --weight_path weight/best.pt --gpu_id 0 --eval --mode val`后得到的mAP为
![image](https://user-images.githubusercontent.com/32457231/103716328-b52ede80-4ffd-11eb-933f-34903cb0cb4a.png)
个人理解是，train和evaluate都是在test_annotation.txt这个文件基础上完成的，但是不太明白为何保存下来的模型best.pt的mAP有这样的区别。若答主有时间，还麻烦您帮忙解释一下

**Comment by jingtianyilong** [comment]
Setting is actually different.
In ·eval_voc.py· we got:
```
APs_voc(self.__multi_scale_val, self.__flip_val)
```
`self.__multi_scale`_ `val, self.__flip_val` are all `False` in defalt config file.

But in train we got:
```
APs_voc()
```
Default value are all false.

**Comment by April4lu** [comment]
Yes, you are right. After setting them to False, the mAPs of both are equal. 
![image](https://user-images.githubusercontent.com/32457231/103720051-6dac5080-5005-11eb-94ac-6ed7a556ed92.png)
And the inference time decreases from 148ms to 16ms. Thank you for your reply!

**Comment by kuaileyuandi** [comment]
你好，可以将你的代码共享一下吗？由于我运行博主代码，遇到bug难以解决。邮箱：1773522147@qq.com,非常感谢

**Comment by April4lu** [comment]
> 你好，可以将你的代码共享一下吗？由于我运行博主代码，遇到bug难以解决。邮箱：[1773522147@qq.com](mailto:1773522147@qq.com),非常感谢

请问是遇到什么样的问题呢？可以交流一下

**Comment by diaodeyi** [comment]
运行eval_voc 提示没有 log_voc_val.txt 请问怎么解决

**Comment by jingtianyilong** [comment]
> 运行eval_voc 提示没有 log_voc_val.txt 请问怎么解决

Get folder's ownership before you run the code.
